_lifeofinternatpuppet/post_13.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title: "Week 13"
+author: Disha-maker
+categories:
+  - team
+  - basics
+tags:
+  - week 13
+---
+
+Hi all!
+
+I had a very busy week with lots going on....
+
+Some of the key highlights were:-
+
+- concat module testing was done (review written and shared with the team later in the week.)
+The testing for this module has been parked for now
+- backlog work assigned - fix layout of hyperlinks on the on the Life of an Intern Blog Page during the free time.
+- Christmas Holidays applied! Looking forward to them!
+- UK Respectful Workplace Training Completed!
+- tag and release done on the haproxy module (v4.4.0)
+- recap session with mentor Paula on release process, the forge and mergeback.
+Thanks Paula for the useful session!
+- litmus testing on IAC Modules (related to the release of the new CI - Google Cloud Platform)
+  Some of the modules tested to date included:-
+  - puppetlabs-apt [link here](https://github.com/puppetlabs/puppetlabs-apt)
+  - puppetlabs-motd [link here](https://github.com/puppetlabs/puppetlabs-motd)
+  - puppetlabs-accounts [link here](https://github.com/puppetlabs/puppetlabs-accounts)
+  - puppetlabs-java [link here](https://github.com/puppetlabs/puppetlabs-java)
+  - puppetlabs-inifile [link here](https://github.com/puppetlabs/puppetlabs-inifile)
+  - puppetlabs-ntp [link here](https://github.com/Disha-maker/puppetlabs-ntp)
```

Comment:
One suggestion here, can you add the links to the module names?  like this:  [puppetlabs-ntp](https://github.com/Disha-maker/puppetlabs-ntp)

Reply:
This is how is looking currently : puppetlabs-ntp [link here](https://github.com/Disha-maker/puppetlabs-ntp)